src/query/functions/temporal/aggregation.go

```diff
@@ -78,6 +82,30 @@ func (a aggProcessor) Init(op baseOp, controller *transform.Controller, opts tra
 	}
 }
 
+// NewQuantileOp create a new base temporal transform for quantile_over_time func
+func NewQuantileOp(args []interface{}, optype string) (transform.Params, error) {
+	if optype == QuantileType {
```

Comment:
If this is creating a `QuantileOp`, why does it need `optype` as input? Anything other than `QuantileType` is an error, so it doesn't seem worth having the argument (fwiw, `HoltWinters` doesn't take one).